indico/modules/events/registration/templates/display/regform_display.html

```diff
@@ -85,7 +85,6 @@
             <div nd-reg-form
                 conf-id="{{ event.id }}"
                 conf-sections="{{ sections | tojson | forceescape }}"
-                conf-currency="{{ currency }}"
```

Comment:
Are you sure that's unused? It's copied to `scope.currency` and e.g. `BillableCtrl` has this code:

            str += ' {0} {1}'.format(item.price, $scope.currency);